### ISSUE TITLE
Replace `crypt` calls with `password_hash`

### ISSUE THREAD

**Issue by JimMackin** [salesagility/SuiteCRM#1197]
SuiteCRM currently uses calls to `crypt` for passwords. The `password_hash` function in PHP 5.5 provides the same functionality but simplifies some of the complexity such as generating a good salt and choosing an appropriate algorithm.

We should seek to use `password_hash` in the next version of SuiteCRM that supports 5.5 and upward (we could even add a version check if we wish to remain backwards compatible).

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Replace `crypt` calls with `password_hash`](https://trello.com/c/uyjoqH7p/109-replace-crypt-calls-with-password-hash)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
